docs/source/jit_language_reference_statements.rst

```diff
@@ -0,0 +1,389 @@
+
+The ``assert`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    assert_stmt ::=  "assert" expression ["," expression]
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+
+* Assert statements in TorchScript do not support ``try\finally``.
+
+The ``return`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    return_stmt ::=  "return" [expression_list]
+    `expression_list <https://docs.python.org/3/reference/expressions.html#grammar-token-expression-list>`
+
+* Return statements in TorchScript do not support ``try\except\finally``.
+
+The ``del`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    del_stmt ::=  "del" target_list
+    `target_list <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-target-list>`
+
+The ``pass`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    pass_stmt ::= "pass"
+
+The ``print`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    print_stmt ::= "print"
+
+The ``break`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    break_stmt ::= "break"
+
+The ``continue`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    continue_stmt ::= "continue"
+
+The ``import`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    import_stmt     ::=  "import" module ["as" identifier] ("," module ["as" identifier])*
+                     | "from" relative_module "import" identifier ["as" identifier]
+                     ("," identifier ["as" identifier])*
+                     | "from" relative_module "import" "(" identifier ["as" identifier]
+                     ("," identifier ["as" identifier])* [","] ")"
+                     | "from" module "import" "*"
+    module          ::=  (identifier ".")* identifier
+    relative_module ::=  "."* module | "."+
+    `identifier_list <https://docs.python.org/3/reference/lexical_analysis.html#grammar-token-identifier>`
+
+The ``future`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    future_stmt ::=  "from" "__future__" "import" feature ["as" identifier]
+                 ("," feature ["as" identifier])*
+                 | "from" "__future__" "import" "(" feature ["as" identifier]
+                 ("," feature ["as" identifier])* [","] ")"
+    feature     ::=  identifier
+    `identifier <https://docs.python.org/3/reference/lexical_analysis.html#grammar-token-identifier>`
+
+
+Compound Statements
+~~~~~~~~~~~~~~~~~~~
+
+The following section describes the syntax of Compound statements that are supported in TorchScript.
+The section also highlights how Torchscript differs from regular Python statements.
+It is modeled after `the compound statements chapter of the Python language reference <https://docs.python.org/3/reference/compound_stmts.html>`_.
+
+The ``if`` statement:
+^^^^^^^^^^^^^^^^^^^^^
+* Torchscript supports both basic ``if/else`` and ternary ``if/else``.
+
+Basic ``if/else`` statement:
+""""""""""""""""""""""""""""
+
+::
+    if_stmt ::=  "if" assignment_expression ":" suite
+             ("elif" assignment_expression ":" suite)
```

Comment:
This seems inaccurate, the "elif" clause can repeat for arbitrary number of times

Reply:
True. I wasn't sure how it could be represented here. I have now added a statement to reflect this